packages/components/stories/Dialog.css

```diff
@@ -0,0 +1,4 @@
+div[role="dialog"] {
```

Comment:
Good idea, but I'm not an huge fan of adding CSS next to our stories.

Just add this JSX in _packages/components/stories/Dialog.js_
```javascript
storiesOf('Dialog', module)
	.addDecorator(story => (
		<div>
			<style>{"\
				.modal, .modal-backdrop {\
					pointer-events: none;\
				}\
				.modal-dialog {\
				    pointer-events: auto;\
			    }\
			"}</style>
			{story()}
		</div>
	))
```

Reply:
CSS in JSX looks really ugly :thinking:

Reply:
Except if @glorieux does not like this.

Reply:
BTW, @rmannibucau, my change request concerns the CSS and not where we have to put it, I can't click on modal buttons anymore with yours!

Reply:
done